refarch-gateway/pom.xml

```diff
@@ -131,20 +129,11 @@
     </dependencies>
 
     <build>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <filtering>true</filtering>
-            </resource>
-        </resources>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven-surefire-plugin.version}</version>
-                <configuration>
-                    <argLine>${jvm.options}</argLine>
-                </configuration>
             </plugin>
             <plugin>
```

Comment:
Maybe its possible to remove this plugin completely as its part of the parent as well? What does the exclusion for lombok achieve?

Reply:
As far as i saw the plugin isn't included in the parent.
And i refactored the lombok inclusion. The exclusion was generated by Spring Initializer bus isn't needed as fas as i could say.